pandas/core/generic.py

```diff
@@ -1940,6 +1940,84 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n = None, frac = None , replace = False, weights = None, seed = None):
+        """
+        Returns a sample of rows from object. 
+    
+        Note that .sample() (with no arguments) will return five random rows.
+    
+        Parameters
+        ----------
+            n: Number of rows to return. Cannot be used with frac.
+               Default = 5 if frac = None. 
+            frac: share of rows to return. Cannot be used with n. 
+            replace {True, False}: Sample with or without replacement.
+            weights: Accepts a Series, list, or ndarray of weights. If called on a 
+                    DataFrame, will also accept the name of a column as a string. 
+                    Must be same length as index.  
+                    Default 'None' results in equal probability weighting.
+            seed: seed to be fed to numpy random.RandomState() Function. Default None. 
+        -----------
+        Note: sample sizes are passed to numpy.random.choice(), which accepts negative values. 
```

Comment:
I don't think this is particularly desirable behavior to copy. It's confusing and not necessary. Instead, I would raise an error in such cases.